store/kv.go

```diff
@@ -29,6 +28,11 @@ func NewDefaultKVStore(rootDir, dbPath, dbName string) KV {
 	}
 }
 
+// NewDefaultKVStore creates instance of default key-value store.
+func NewDefaultKVStore(rootDir, dbPath, dbName string) KV {
+	return NewKVStore(rootDir, dbPath, dbName, false)
```

Comment:
should we change it here to `true`? 
also why is this `NewDefaultKVStore` necessary? vs just calling `NewKVStore`?

Reply:
changed
`NewDefaultKVStore` used in tests and localDb